egs/librispeech/ASR/distillation_with_hubert.sh

```diff
@@ -0,0 +1,95 @@
+stage=$1
+export CUDA_VISIBLE_DEVICES="2,3,4,5"
```

Comment:
Not all users have at least 6 GPUs. Please remove this line.

Reply:
An example is added.
A user MUST set this variable.
CUDA_VISIBLE_DEVICES is needed by CodebookIndexExtractor do determine number of jobs running parallel.